checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -154,6 +154,13 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFbc) {
                 org.checkerframework.checker.nullness.compatqual.MonotonicNonNullType.class,
                 MONOTONIC_NONNULL);
 
+        // Alias deprecated annotation
```

Comment:
This should be in the KeyFor Checker.  It's also not copying the values in `@Covariant`, so it's causes crashes:
https://travis-ci.org/typetools/checker-framework/jobs/251223815#L1378